app/services/plans/update_service.rb

```diff
@@ -126,8 +126,16 @@ def cascade_charge_update(charge, payload_charge)
       plan.children.includes(:charges).find_each do |p|
         child_charge = p.charges.find { |c| c.parent_id == charge.id }
 
-        if child_charge && charge.equal_properties?(child_charge)
-          Charges::UpdateJob.perform_later(charge: child_charge, params: payload_charge, cascade: true)
+        if child_charge
+          Charges::UpdateJob.perform_later(
+            charge: child_charge,
+            params: payload_charge,
+            options: {
+              cascade: true,
+              parent_filters: charge.filters.map(&:attributes),
+              equal_properties: charge.equal_properties?(child_charge)
```

Comment:
Maybe we should rename the option attribute `cascade_options`, to clearly defined what is its purpose.